python/ray/serve/generated/serve_pb2.py

```diff
@@ -2,12 +2,13 @@
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Non-blocker: This is totally fine to edit. But maybe better just skip running linter on it. Also, it should only be checked in until we dropped Python 3.7 support so doesn't matter that much

Reply:
oops, I don't want to update this file too, let me skip this file.